js/lesson4Assignment2.js

```diff
@@ -0,0 +1,18 @@
+function dogPhotoClick(dogName, dogBreed, adoptionFees) {
+
+  alert(dogName + ' is a(n) ' + dogBreed + '. Adoption cost is $' + adoptionFees + '.');
```

Comment:
Nice job, You can also use template literals for string concatenation: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals